plugins/vue-filters.js

```diff
@@ -28,3 +28,17 @@ Vue.filter('optimisedImageUrl', (imageUrl, contentType) => {
   }
   return imageUrl;
 });
+
+Vue.filter('inCurrentLanguage', (val, currentLocale) => {
```

Comment:
This needs to account for the fact that the keys of `val` may be three-letter codes.

For instance, on the record /2023813/o_12654, the title is in French but with the code `fra` so it is not detected or rendered when viewing the site at /fr.

I think we will therefore need to do something like:
* Extend the locale definitions in /plugins/i18n/locales.json to also include the three-letter codes in a new property named `isoAlpha3`
* In this filter, find the three-letter code for `currentLocale` and also prepend that onto `languageKeys`

Reply:
I have updated the locales and the filter so it accepts 2 and 3 character codes.